Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -1676,4 +1678,105 @@ M  END
     /// SMARTS-based queries are not written for these:
     CHECK(molb.find("V    ") == std::string::npos);
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE("read metadata from PNG", "[reader][PNG]") {
```

Comment:
Should there be one or two cases of malformed PNGs to test header errors?

Reply:
I added a truncated example below. Is there something in particular that we should be testing for?

Reply:
No, just the code path.